### DISCUSSION TITLE
EOS placed REVENUE SELL order???

### DISCUSSION THREAD

**Discussion by IHateNameStealers** [TeamWertarbyte/crypto-trading-bot#31] [Q&A] [answered]
Just noticed in my log the following.

EOS placed REVENUE SELL order for 0.3177976126013462 USD.

What does this message mean?

**Comment by saschb2b** [comment, marked as answer]
A revenue sell is everything above your initial investment.
E.g. each currency gets bought with `50$`. Once it's above this value it tries to sell that. 
In your case `EOS` just made you 0.32$ profit

**Reply by IHateNameStealers** [reply]
Thank you for explaining this!
Why do I not see this back in my Bittrex orders page?
(btw. My amountPerInvest is set to $20)

**Reply by saschb2b** [reply]
Then the actual order could not be fullfilled. The bot tried to sell, hence `placed`, but nobody bought it sadly.
The sell orders are `fill or kill`meaning they either get bought directly from else or aborted.
Seems in your case it just got aborted.
But don't worry. The bot will try it next round again until someone buys it from you. (then it's visible on bittrex as well)
I could extent the logging and or the wording if that could help.
Any feedback helps
